pkg/controller/controller.go

```diff
@@ -501,7 +501,7 @@ func (c *Controller) reconcile(ctx context.Context, req reconcile.Request, obj i
 		}
 	}
 
-	ctx = setLoggerCtxValue(ctx, loggerKeyObject, m.GetSelfLink())
+	ctx = setLoggerCtxValue(ctx, loggerKeyObject, fmt.Sprintf("%s/%s", m.GetNamespace(), m.GetName()))
```

Comment:
We lose the type information here which is a shame but its also less verbose in the logs.